hudi-spark-datasource/hudi-spark3-extensions_2.12/src/test/scala/org/apache/spark/sql/hudi/TestDelete.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.hudi
+
+import org.apache.hadoop.fs.Path
+
+class TestDelete extends TestHoodieSqlBase {
+
+  test("Test Delete Table") {
+    withTempDir { tmp =>
+      val tablePath = new Path(tmp.toString, "deleteTable").toUri.toString
+      spark.sql("set spark.hoodie.shuffle.parallelism=4")
+      spark.sql(
+        s"""create table deleteTable (
+           |keyid int,
+           |name string,
+           |price double,
+           |col1 long,
+           |p string,
+           |p1 string,
+           |p2 string) using hudi
+           |partitioned by (p,p1,p2)
+           |options('hoodie.datasource.write.table.type'='MERGE_ON_READ',
+           |'hoodie.datasource.write.precombine.field'='col1',
```

Comment:
here in base sql implement, it called `versionColumn`, should align with this?

Reply:
agree